test/_stub/AbstractSpec.scala

```diff
@@ -0,0 +1,9 @@
+package _stub
+
+import org.scalatest.{FunSpec, Matchers}
+import org.scalatestplus.mockito.MockitoSugar
+import org.scalatestplus.play.guice.GuiceOneAppPerTest
+
+abstract class AbstractSpec extends FunSpec with GuiceOneAppPerTest with MockitoSugar with Matchers {
+
+}
```

Comment:
This is the base class for every unit test classes. This class extends to various classes that are helpful for testing.